src/components/organisms/CompareModal/ComparisonList.tsx

```diff
@@ -0,0 +1,103 @@
+import {useCallback} from 'react';
+
+import {Button, Checkbox, Col} from 'antd';
+
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {
+  comparisonToggled,
+  diffViewOpened,
+  selectComparisonListItems,
+  selectIsComparisonSelected,
+} from '@redux/reducers/compare';
+
+import * as S from './ComparisonList.styled';
+
+export type ComparisonListItem = HeaderItemProps | ComparisonItemProps;
+
+export const ComparisonList: React.FC = () => {
+  const items = useAppSelector(state => selectComparisonListItems(state.compare));
+
+  return (
+    <div>
+      {items.map(item => {
+        return item.type === 'header' ? (
+          <HeaderItem key={item.kind} {...item} />
+        ) : (
+          <ComparisonItem key={item.id} {...item} />
+        );
+      })}
+    </div>
+  );
+};
+
+type HeaderItemProps = {
+  type: 'header';
+  kind: string;
+  count: number;
+};
+
+function HeaderItem({kind, count}: HeaderItemProps) {
+  return (
+    <S.HeaderRow key={kind}>
+      <Col span={11}>
+        <S.Title>
+          {kind} <S.ResourceCount>{count}</S.ResourceCount>
+        </S.Title>
+      </Col>
+
+      <Col span={2} />
+
+      <Col span={11}>
+        <S.Title>
+          {kind} <S.ResourceCount>{count}</S.ResourceCount>
+        </S.Title>
+      </Col>
+    </S.HeaderRow>
+  );
+}
+
+type ComparisonItemProps = {
+  type: 'comparison';
+  id: string;
+  namespace: string;
+  name: string;
+  leftActive: boolean;
+  rightActive: boolean;
+  canDiff: boolean;
+};
+
+function ComparisonItem({id, namespace, name, leftActive, rightActive, canDiff}: ComparisonItemProps) {
```

Comment:
I think across the project we destructure the props inside the component, maybe we should stay consistent